test/Common.cpp

```diff
@@ -104,6 +104,7 @@ void CommonOptions::addOptions()
 {
 	options.add_options()
 		("evm-version", po::value(&evmVersionString), "which evm version to use")
+		("eof-version", po::value(&eofVersionString)->default_value("")->implicit_value(""), "which eof version to use")
```

Comment:
Is `""` even a valid value? Why not match what we do for solc and use `count()` to see if the option was given or not?
```suggestion
		("eof-version", po::value(&eofVersionString)->implicit_value("1"), "which eof version to use")
```